coresimd/x86/sse.rs

```diff
@@ -968,6 +968,14 @@ pub unsafe fn _mm_setzero_ps() -> __m128 {
     __m128(0.0, 0.0, 0.0, 0.0)
 }
 
+/// A utility function for creating masks to use with Intel shuffle and permute intrinsics.
+#[inline]
+#[allow(non_snake_case)]
+#[stable(feature = "simd_x86", since = "1.27.0")]
```

Comment:
This will probably need to be 1.28.0 since 1.27 is in beta already and I don't think this will be back ported to beta.

Reply:
Done